File: project/backend/src/main/java/co/edu/utp/isc/gia/examsapp/web/dto/MultiQuestionDto.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package co.edu.utp.isc.gia.examsapp.web.dto;

import co.edu.utp.isc.gia.examsapp.web.dto.abstractdto.QuestionDto;
import java.util.List;
import lombok.Getter;
import lombok.Setter;
import lombok.experimental.SuperBuilder;

/**
 *
 * @author CJ
 */

@Getter @Setter
@SuperBuilder()
public class MultiQuestionDto extends QuestionDto{
    
    private List<AnswerOptionDto> answerOptions;
}

File: project/frontend/src/app/dto/MultiQuestionDto.ts
import {OnInit } from '@angular/core';
import { QuestionDto } from './abstractDto/QuestionDto';
import { AnswerOptionDto } from './AnswerOptionDto';

export class MultiQuestionDto extends QuestionDto implements OnInit {
    
    public answerOptions : Array<AnswerOptionDto>;

    ngOnInit() {
        this.answerOptions = new Array<AnswerOptionDto>();
    }

}
